src/Microsoft.Health.Fhir.Shared.Api/Features/Formatters/GenericXmlOutputFormatter.cs

```diff
@@ -0,0 +1,27 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using EnsureThat;
+using Hl7.Fhir.Model;
+using Microsoft.AspNetCore.Mvc.Formatters;
+
+namespace Microsoft.Health.Fhir.Api.Features.Formatters
+{
+    public class GenericXmlOutputFormatter : XmlSerializerOutputFormatter
```

Comment:
Should we rename it to 'NonFhirResourceXmlOutputFormatter'?

Reply:
Thats the original name I had :P